benchmark/README.md

```diff
@@ -31,8 +33,145 @@ npm run benchmark -- -u 4 -e 15 -r 5
 - `npm run benchmark` to run the benchmark with the default setting, to see the available flags run
 - `npm run benchmark -- --help`
 
-## Comming soon
+## Custom Tasks
+
+DiscoJS currently provides several pre-define popular tasks such as titanic, simple-face and cifar10. In order
+to understand how to add your own custom task, we will go over how we added simple-face to discojs.
+
+### Task
+
+The `Task` class is the first piece of the puzzle, this contains all the crucial information from training to mode.
+
+The task class for simple face can be found in [discojs/src/tasks/simple_face.ts](../discojs/src/tasks/simple_face.ts),
+and the contents look as follows. (For brevity we have replaced some lines with ...).
+
+Note that the preprocessFunctions are not yet supported via the Task object, we will explain after how to add a it
+via the data loader.
+
+After exporting the Task, you need to also export a function called `model` that returns the layers model. If you use a 
+pre-trained model, you can simply load and return said model in the function via `tf.loadLayersModel(modelPath)`.
+
+```js
+import * as tf from '@tensorflow/tfjs'
+
+import { Task } from '../task'
+
+export const task: Task = {
+  taskID: 'simple_face',
+  displayInformation: {
+    taskTitle: 'Simple Face',
+    summary: 'Can you detect if the person in a picture is a child or an adult?',
+    ...
+  },
+  trainingInformation: {
+    modelID: 'simple_face-model',
+    epochs: 50,
+    roundDuration: 1,
+    validationSplit: 0.2,
+    batchSize: 10,
+    preprocessFunctions: [],
+    learningRate: 0.001,
+    modelCompileData: {
+      optimizer: 'sgd',
+      loss: 'categoricalCrossentropy',
+      metrics: ['accuracy']
+    },
+    dataType: 'image',
+    csvLabels: false,
+    IMAGE_H: 200,
+    IMAGE_W: 200,
+    LABEL_LIST: ['child', 'adult'],
+    scheme: 'Federated'
+  }
+}
+
+export function model (): tf.LayersModel {
+  // Init model
+  const model = tf.sequential()
+
+  // Add layers
+  model.add(...)
+
+  return model
+```
+
+After adding the `simple_face.ts` task class, we also need to export it in the `index.ts` file which lives in the same folder.
+
+The contents of `index.ts` are as follows
+
+```js
+export * as cifar10 from './cifar10'
+export * as lus_covid from './lus_covid'
+export * as mnist from './mnist'
+export * as simple_face from './simple_face' // <---- 
+export * as titanic from './titanic'
+```
+
+This allows us to access our task as via discojs:
```

Comment:
unclear what you here mean by access. i guess here is the point where one switches to now either use the new task in browser or in node (we can explain both). maybe also remind that one has to first build sth again, till it's usable